impeller/renderer/backend/vulkan/texture_vk.h

```diff
@@ -44,9 +44,20 @@ class TextureVK final : public Texture, public BackendCast<TextureVK, Texture> {
 
   bool IsSwapchainImage() const { return source_->IsSwapchainImage(); }
 
+  // These methods should only be used by render_pass_vk.h
+  void SetFramebuffer(const SharedHandleVK<vk::Framebuffer>& framebuffer);
```

Comment:
Would it make sense to save this on the RenderTarget, not the Texture?  This isn't applicable to every Texture, right?

Reply:
The RenderTarget itself is const/immutable whereas the Texture is stateful.

Reply:
That's not a problem though.  We can make the frame buffer and render pass when we create the RenderTarget.  We shouldn't ever have a RenderTarget without a render pass and it will never change, right?

Reply:
Unfortunately that is all constructed above the backend layer, and there are no metal or gles analogs for these objects.